lib/config/presets/internal/replacements.ts

```diff
@@ -940,7 +940,7 @@ const messageFormat: PresetTemplate = {
       replacementVersion: '5.0.0',
     },
   ],
-  title: 'messageFormat-{{package}}-to-@messageformat/{{package}}',
+  title: 'messageFormat-package-to-@messageformat/package',
```

Comment:
```suggestion
  title: 'messageFormat-to-scoped',
```
we should use same name format as for other scoped packages.

this needs to be done in a separate PR.